add-on/src/options/options.html

```diff
@@ -1,56 +1,167 @@
     </style>
+    <script src="../lib/npm/browser-polyfill.min.js"></script>
   </head>
-
   <body>
-
     <form>
-      <fieldset style="line-height: 2em; font-family: sans-serif;">
-        <legend>Basic</legend>
-
-
-        <p><label>Public Gateways Hosts <input type="text" id="publicGateways" size="40" required placeholder="List of hostnames separated with spaces"/></label></p>
-
-        <p><label>Custom Gateway URL <input type="url" id="customGatewayUrl" size="40" inputmode="url" required pattern="https?://.+"/></label></p>
-
-
-        <p><label>IPFS API URL <input type="url" id="ipfsApiUrl" size="40" inputmode="url" required pattern="https?://.+"/></label></p>
-
-        <p><label>Redirect IPFS Requests to the Custom Gateway  <input type="checkbox" id="useCustomGateway" /></label></p>
-
-        <p><label>Automatic Mode <small>(Redirect only when IPFS API is Online and node has peers)</small> <input type="checkbox" id="automaticMode" /></label></p>
-
+      <fieldset>
+        <legend>Gateways</legend>
+        <div>
+          <label for="publicGateways">
+            <dl>
+              <dt>Public Gateways</dt>
+              <dd>list of hostnames known for exposing IPFS resources under <code>/ipfs/</code> path</dd>
+            </dl>
+          </label>
+          <textarea id="publicGateways" rows="4" required placeholder="List of hostnames separated with spaces or newlines"></textarea>
+        </div>
+        <div>
+          <label for="customGatewayUrl">
+            <dl>
+              <dt>Custom Gateway</dt>
+              <dd>URL address of preferred HTTP2IPFS&nbsp;Gateway</dd>
+            </dl>
+          </label>
+          <input type="url" id="customGatewayUrl" inputmode="url" required pattern="https?://.+"/>
+        </div>
+        <div>
+          <label for="useCustomGateway">
+            <dl>
+              <dt>Use Custom Gateway</dt>
+              <dd>replace requests to <em>Public</em> gateway with <em>Custom</em> one</dd>
+            </dl>
+          </label>
+          <input type="checkbox" id="useCustomGateway" />
+        </div>
       </fieldset>
     </form>
-
     <form>
-      <fieldset style="line-height: 2em; font-family: sans-serif;">
-        <legend>Advanced</legend>
-
-        <p><label>IPFS API Status Poll Interval (ms) <input type="number" inputmode="numeric" id="ipfsApiPollMs" min="1000" max="60000" step="1000" required /></label></p>
-
+      <fieldset>
+        <legend>API</legend>
+        <div>
+          <label for="ipfsApiUrl">
+            <dl>
+              <dt>IPFS API URL</dt>
+              <dd>hint: this is where <code>/webui</code> lives</dd>
+            </dl>
+          </label>
+          <input type="url" id="ipfsApiUrl" inputmode="url" required pattern="https?://.+"/>
+        </div>
+        <div>
+          <label for="ipfsApiPollMs">
+            <dl>
+              <dt>Status Poll Interval</dt>
+              <dd>how often peer count is refreshed (in miliseconds)</dd>
```

Comment:
from AndyS2:
> Time interval after which peer count is refreshed (in milliseconds)

Reply:
I believe "How often peer count is refreshed" is fine, use simpler words whenever it makes sense.